CMakeLists.txt

```diff
@@ -23,34 +23,34 @@ endif()
 # Set policy CMP0057 to support IN_LIST operators
 cmake_policy(SET CMP0057 NEW)
 
-# MIOpen dialect.
-set(MLIR_MIOPEN_DRIVER_ENABLED 1 CACHE BOOL "Enable build MIOpen driver")
-set(MLIR_MIOPEN_DRIVER_E2E_TEST_ENABLED 0 CACHE BOOL "Enable build E2E tests for MIOpen driver")
-set(MLIR_MIOPEN_DRIVER_MISC_E2E_TEST_ENABLED 0 CACHE BOOL "Enable build miscellaneous E2E tests for MIOpen driver")
-set(MLIR_MIOPEN_DRIVER_PR_E2E_TEST_ENABLED 1 CACHE BOOL "Enable build PR-triggered E2E tests for MIOpen driver")
-set(MLIR_MIOPEN_DRIVER_RANDOM_DATA_SEED "none" CACHE STRING "Enable E2E tests using random data")
-set(MLIR_MIOPEN_DRIVER_XDLOPS_TEST_ENABLED 0 CACHE BOOL "Enable build E2E tests for XDLops")
-set(MLIR_MIOPEN_DRIVER_TEST_GPU_VALIDATION 1 CACHE BOOL "Enable E2E tests with GPU validation")
-set(MLIR_MIOPEN_SQLITE_ENABLED 0 CACHE BOOL "Enable SQLite integration")
-set(MLIR_ENABLE_SQLITE "${MLIR_MIOPEN_SQLITE_ENABLED}")
+# Rock dialect.
+set(MLIR_ROCK_DRIVER_ENABLED 1 CACHE BOOL "Enable build Rock driver")
```

Comment:
I think the driver should be `rocmlir-driver` and these vsrisbles should be rensmed similarly

Reply:
Still thinking about this.. I may just create a 'rock-driver' and a 'xmir-driver' to untangle the complexity in mlir-miopen-driver.

Reply:
I  don't agree. `mlir-miopen-driver` is basically a wrapper around `mlir-opt`, and so it should be `rocmlir-driver` by analogy with `rocmlir-opt`